sdk/ml/azure-ai-ml/dev_requirements.txt

```diff
@@ -13,4 +16,7 @@ pytest
 pydash
 pytest-vcr
 azure-mgmt-storage~=19.1.0
-pywin32==227 ; sys_platform == 'win32'
+azure-devtools==1.2.0
```

Comment:
Install local version of azure-devtools.

Reply:
Left comment with suggestion of fix. you already have the correct relative install present on line 3.